packages/jest-runtime/src/helpers.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import chalk from 'chalk';
+
+export const enhanceUnexpectedTokenMessage = (e: Error) => {
+  e.stack =
+    `${chalk.bold.red('Jest encountered an unexpected token')}
+
+This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not JavaScript.
+
+Jest runs on source files, so it happens that it needs to process them with a transformer (${chalk.bold(
+      '"transform"',
+    )} option).
+By default a transformer is not processing "node_modules" for performance reasons.
+If you need to process some of your node modules please adjust the ${chalk.bold(
+      '"transformIgnorePatterns"',
+    )} option.
+
+If you need to read non-JS files (e.g. binary assets), stubbing them out with ${chalk.bold(
+      '"moduleNameMapper"',
+    )} should help.
```

Comment:
How about: 

> Jest encountered an unexpected token
>
> This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.
>
> By default, if Jest sees a `.babelrc`, it will use that to transform your files, ignoring `node_modules`. To have some, of your `node_modules` files transformed, you can specify a custom "transformIgnorePatterns" in your config.
>
> If you need a custom transformation specify a "transform" option in your config.
>
> If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.